pmd-core/src/main/java/net/sourceforge/pmd/lang/ast/impl/antlr4/AntlrToken.java

```diff
@@ -1,8 +1,8 @@
-/**
+/*
  * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
  */
 
-package net.sourceforge.pmd.cpd.token;
+package net.sourceforge.pmd.lang.ast.impl.antlr4;
```

Comment:
I described this [here](https://github.com/pmd/pmd/pull/2055#issuecomment-538788501)

So `ast.impl.antlr4` would be a package relevant for language implementors to use, while  `ast.internal` should not be used outside of pmd-core, and `ast` is relevant to everyone.

I'm not sure it's sensible to deprecate on master, since the API will still be here and easy to reach. Maybe we should mention this in the 7.0 release notes though